x/gov/legacy/v034/types.go

```diff
@@ -1,513 +0,0 @@
-package v034
```

Comment:
Will be harder to follow the history of the files if we delete and re-create instead or move files